web/cobrands/bristol/assets.js

```diff
@@ -84,4 +122,28 @@ fixmystreet.assets.add(options, {
     filter_value: ['PT01', 'PT02', 'PT03']
 });
 
+fixmystreet.assets.add(options, {
+    asset_category: "Flooding/Gully/Drainage",
```

Comment:
Should the Flooding (FRST) asset layer be removed? or should FRST be included in the filter_value?

Reply:
The council are retiring that mapping...I've removed it